src/entities/BaseEntity.java

```diff
@@ -0,0 +1,63 @@
+package entities;
+
+import java.time.ZonedDateTime;
+
+public abstract class BaseEntity {
+    private Object id;
+    private Object updatedby;
```

Comment:
Maybe you could use generics here? I don't this is a good idea because you would never know what is this and could become a headache for other developers to maintain it.